backend/database/migrations/2023_07_07_023101_add_account_number_to_accounts_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddAccountNumberToAccountsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('accounts', function (Blueprint $table) {
+            $table->string('account_number')->default('00000000000');
```

Comment:
make this nullable and unique, dont put default so that we can have unique constraint